app/admin/sectors.rb

```diff
@@ -58,6 +60,7 @@
 
     f.inputs do
       f.input :name
+      f.input :cp_unit, label: 'Carbon Performance benchmark unit'
```

Comment:
should we just call it `Carbon Performance Unit` ?

Reply:
year, works for me :) @simaob ?